### PR TITLE
docs: add kedro-partitioned plugin

### PR DESCRIPTION
**Pull request by sbrugman**
## Description
Missing this plugin in the docs

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1854"></a>

### CODE REVIEW

File: docs/source/extend_kedro/plugins.md [hunk 1]
```diff
@@ -224,3 +224,4 @@ Your plugin needs to have an [Apache 2.0 compatible license](https://www.apache.
 - [kedro-airflow-k8s](https://github.com/getindata/kedro-airflow-k8s), by [GetInData](https://github.com/getindata), enables running a Kedro pipeline with Airflow on a Kubernetes cluster
 - [kedro-vertexai](https://github.com/getindata/kedro-vertexai), by [GetInData](https://github.com/getindata), enables running a Kedro pipeline with Vertex AI Pipelines service
 - [kedro-azureml](https://github.com/getindata/kedro-azureml), by [GetInData](https://github.com/getindata), enables running a Kedro pipeline with Azure ML Pipelines service
+- [kedro-partitioned](https://github.com/ProjetaAi/kedro-partitioned), by [Gabriel Daiha Alves](https://github.com/gabrieldaiha) and [Nickolas da Rocha Machado](https://github.com/nickolasrm], extends the functionality on processing partitioned data.
```
Comment (deepyaman) [inline docs/source/extend_kedro/plugins.md:227]:
```suggestion
- [kedro-partitioned](https://github.com/ProjetaAi/kedro-partitioned), by [Gabriel Daiha Alves](https://github.com/gabrieldaiha) and [Nickolas da Rocha Machado](https://github.com/nickolasrm], extends the functionality for processing partitioned data
```

**Review by deepyaman — CHANGES_REQUESTED** [review]
Thanks for the PR! Minor suggestion, but aside from that, do you mind:

1. Switching the PR title to "Add `kedro-partitioned` to list of community plugins" or something (we don't use conventional commit, for better or for worse, so I wouldn't want this one to look different)
2. Fixing DCO (the instructions are there in the failed check)

File: docs/source/extend_kedro/plugins.md (hunk 1 above)
Comment (MerelTheisenQB) [inline docs/source/extend_kedro/plugins.md:227]:
```suggestion
- [kedro-partitioned](https://github.com/ProjetaAi/kedro-partitioned), by [Gabriel Daiha Alves](https://github.com/gabrieldaiha) and [Nickolas da Rocha Machado](https://github.com/nickolasrm), extends the functionality on processing partitioned data.
```

**Review by MerelTheisenQB — APPROVED** [review]
Thanks for adding this! @sbrugman 👍

**Comment by stichbury** [comment]
@sbrugman  I'd like to take your suggested changes into the docs once we make the new release -- many apologies that it's taken a while to get to them. 

I can make the changes suggested for you or please could you take a look and make them yourself. I'll assume it's OK for me to proceed and merge the PR if you've not made any further changes by 8th December 2022.

Thank you again, we appreciate your contribution 🙏

**Pull request merged**